arrow-syntax.api

```diff
@@ -0,0 +1,1632 @@
+	public static final fun reverse (Lkotlin/jvm/functions/Function22;)Lkotlin/jvm/functions/Function22;
+	public static final fun reverse (Lkotlin/jvm/functions/Function2;)Lkotlin/jvm/functions/Function2;
+	public static final fun reverse (Lkotlin/jvm/functions/Function3;)Lkotlin/jvm/functions/Function3;
+	public static final fun reverse (Lkotlin/jvm/functions/Function4;)Lkotlin/jvm/functions/Function4;
+	public static final fun reverse (Lkotlin/jvm/functions/Function5;)Lkotlin/jvm/functions/Function5;
+	public static final fun reverse (Lkotlin/jvm/functions/Function6;)Lkotlin/jvm/functions/Function6;
+	public static final fun reverse (Lkotlin/jvm/functions/Function7;)Lkotlin/jvm/functions/Function7;
+	public static final fun reverse (Lkotlin/jvm/functions/Function8;)Lkotlin/jvm/functions/Function8;
+	public static final fun reverse (Lkotlin/jvm/functions/Function9;)Lkotlin/jvm/functions/Function9;
+}
+
+public final class arrow/core/TupleKt {
+	public static final fun plus (Larrow/core/Tuple10;Ljava/lang/Object;)Larrow/core/Tuple11;
+	public static final fun plus (Larrow/core/Tuple11;Ljava/lang/Object;)Larrow/core/Tuple12;
+	public static final fun plus (Larrow/core/Tuple12;Ljava/lang/Object;)Larrow/core/Tuple13;
+	public static final fun plus (Larrow/core/Tuple13;Ljava/lang/Object;)Larrow/core/Tuple14;
+	public static final fun plus (Larrow/core/Tuple14;Ljava/lang/Object;)Larrow/core/Tuple15;
+	public static final fun plus (Larrow/core/Tuple15;Ljava/lang/Object;)Larrow/core/Tuple16;
+	public static final fun plus (Larrow/core/Tuple16;Ljava/lang/Object;)Larrow/core/Tuple17;
+	public static final fun plus (Larrow/core/Tuple17;Ljava/lang/Object;)Larrow/core/Tuple18;
+	public static final fun plus (Larrow/core/Tuple18;Ljava/lang/Object;)Larrow/core/Tuple19;
+	public static final fun plus (Larrow/core/Tuple19;Ljava/lang/Object;)Larrow/core/Tuple20;
+	public static final fun plus (Larrow/core/Tuple20;Ljava/lang/Object;)Larrow/core/Tuple21;
+	public static final fun plus (Larrow/core/Tuple21;Ljava/lang/Object;)Larrow/core/Tuple22;
+	public static final fun plus (Larrow/core/Tuple4;Ljava/lang/Object;)Larrow/core/Tuple5;
+	public static final fun plus (Larrow/core/Tuple5;Ljava/lang/Object;)Larrow/core/Tuple6;
+	public static final fun plus (Larrow/core/Tuple6;Ljava/lang/Object;)Larrow/core/Tuple7;
+	public static final fun plus (Larrow/core/Tuple7;Ljava/lang/Object;)Larrow/core/Tuple8;
+	public static final fun plus (Larrow/core/Tuple8;Ljava/lang/Object;)Larrow/core/Tuple9;
+	public static final fun plus (Larrow/core/Tuple9;Ljava/lang/Object;)Larrow/core/Tuple10;
+	public static final fun plus (Lkotlin/Pair;Ljava/lang/Object;)Lkotlin/Triple;
+	public static final fun plus (Lkotlin/Triple;Ljava/lang/Object;)Larrow/core/Tuple4;
+}
+
+public final class arrow/core/TuplingKt {
+	public static final fun tupled (Lkotlin/jvm/functions/Function10;)Lkotlin/jvm/functions/Function1;
```

Comment:
Deprecate. Not useful since arity-n functions, and tupling is not supported in Kotlin.